_posts/2020-02-01-software-testing-the-seven-truths-you-have-to-deal-with.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title:  "Software testing: the seven truths you have to deal with!"
+author: "Helena Rodrigues"
+date:   2020-02-01
+thumbnail: /img/posts/software-testing-the-seven-truths-you-have-to-deal-with/software-testing.jpeg
+categories: [engineering-quality]
+tags: [testing]
+permalink: /software-testing-the-seven-truths-you-have-to-deal-with/
+---
+
+<center>
+    <img alt="Software testing: the seven truths you have to deal with" src="/img/posts/software-testing-the-seven-truths-you-have-to-deal-with/software-testing.jpeg" style="width: 90%; margin-bottom: 10px;" />
+</center>
+
+## Introduction
+
+Humans developed software and they make it with a purpose. Software will do what it is programmed to do, through coding. However, with the increasing complexity of the systems developed, testing activities have become inevitable and an essential piece in software development.
```

Comment:
Think the first sentence should rather be: "Humans **develop** software and they **do** it with a purpose." WDYT?